plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -69,6 +69,10 @@ function getCmConfig() {
 						value = tiddler.fields.text.trim();
 						config[element] = value;
 					break;
+					case "integer":
+					  value = parseInt(tiddler.fields.text.trim());
+					  config.element = value;
```

Comment:
`element` is a variable not an object element. see code above. it's extracted from the tiddler title.